requirements.txt

```diff
@@ -2,3 +2,4 @@ Django>=1.8,<1.9
 PyYAML
 requests>=2.0
 django-crispy-forms>=1.4.0
+django-countries==3.3
```

Comment:
Can we do `>=3.3,<4.0`?  I don't see a need to hard-code a specific version, since all we need is an implementation that supports their 3.x API.